client/app/mobile/modules/series/root/masterSeries/seriesGrid/SerieCard.js

```diff
@@ -2,25 +2,77 @@ import React from 'react';
 import {
     View,
     Image,
-    Text
+    Text,
+    StyleSheet
 } from 'react-native';
 import Link from '../../../../shared/Link';
 
 const SerieCard = function({ serie }) {
     return(
-        <Link to={{ routeName: 'Detail', params: { id: serie.id }}}  >
-            <View>
+        <Link to={{ routeName: 'Detail', params: { id: serie.id } }}>
+            <View style={ styles.container }>
                 <Image 
-                    style={{ width: 150, height: 150 }}
+                    style={ styles.image }
                     source={{
                         uri: serie.image.medium,
                         cache: 'only-if-cached',
                     }}
                 />
-                <Text>{ serie.title }</Text>
+                <View style={ styles.dataContainer }>
+                    <Text style={ styles.title }>{ serie.title }</Text>
+                    <View style={ styles.dataAlignRow }>
+                        <View style={ styles.leftDataChild }>
+                            <Text style={ styles.textFormat }>{ serie.language }</Text>
+                        </View>
+                        <View style={ styles.rightDataChild }>
+                            <Text style={ styles.textFormat }>Episodios: 100</Text>
+                            <Text style={ styles.textFormat }>Temporadas: 6</Text>
```

Comment:
Very complex. Why using numbers? I think for a pr is better setting the values in variables and then only change them for props variables.

Reply:
I am waiting to have the API to use real data, right now i have not this information in my fakeData